graviti/dataframe/column/series.py

```diff
@@ -56,6 +58,29 @@ def __init__(
         else:
             self._indices_data = {raw_index: location for location, raw_index in enumerate(index)}
             self._indices = list(index)
+    
+    def __repr__(self) -> str:
+        indent = _INDENT * 2
+        length = self.__len__()
+        lines = []
+        if self._indices is None:
+            if length >= _MAX_REPE_NUMBER:
+                repr_indices = range(_MAX_REPE_NUMBER)
+            else:
+                repr_indices = range(length)
+        else:
+            if length >= _MAX_REPE_NUMBER:
+                repr_indices = self._indices[:_MAX_REPE_NUMBER]
+            else:
+                repr_indices = self._indices
+        if self.name:
+            lines.append(f"{_INDENT:<{_MAX_REPE_NUMBER}}{indent}{self.name:<{_MAX_REPE_NUMBER}}")
+        for key,value in zip(repr_indices, self._data):
```

Comment:
your code is not well formatted, it should not happen.
have you enabled the pre-commit local check?